w_common_tools/lib/sass.dart

```diff
@@ -0,0 +1,31 @@
+// Copyright 2019 Workiva Inc.
```

Comment:
Although this entrypoint is public in `w_common`, it appears to be [unused](https://sourcegraph.plat.workiva.net/search?q=w_common%2Fsass.dart). I would vote for not exporting these publicly and instead treating `bin/` as the only public API for this new package.